### PR TITLE
Add Work section and About Myself section

### PR DESCRIPTION
**Pull request by godiarksel**
- Add Work and About Myself section using Flexbox and Grid layout.
- Style Webpage according to Figma design templates.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
+    </div>
+
+    <div class="tonic-container">
+        <div><img src="images/multi-post-portfolio.png" alt="multi-post-portfolio"></div>
+        <div class="tonic-heading">Multi-Post Stories</div>
+        <div class="canopy-container">
```
Comment (mahbubul14) [inline index.html:71]: - [ ] The project title ("Multi-Post Stories") should use a header tag `hx` (Ex: `<h2>`, `<h3>` etc.) :+1:

File: index.html [hunk 2]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+         </ul>
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
+    </div>
+        <div class="tonic-container">
+            <div><img src="images/tonic2.png" alt="tonic"></div>
+            <div class="tonic-heading">Tonic</div>
+            <div class="canopy-container">
```
Comment (mahbubul14) [inline index.html:98]: - [ ] The project title ("Tonic") should use a header tag `hx` (Ex: `<h2>`, `<h3>` etc.) :+1:

File: index.html [hunk 3]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+               </ul>
+            </div>
+            <div class="tonic-btn">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+            <div class="tonic-container">
+                <div><img src="images/multi-post-stories2.png" alt="multi-post-stories2"></div>
+                <div class="tonic-heading">Multi-Post Stories</div>
+                <div class="canopy-container">
```
Comment (mahbubul14) [inline index.html:125]: - [ ] The project title ("Multi-Post Stories") should use a header tag `hx` (Ex: `<h2>`, `<h3>` etc.) :+1:

File: index.html [hunk 4]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
     </ul>
     </div>
 </section>
-  
+<section class="work-section">
+    <div class="tonic-container">
+        <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
+        <div class="tonic-heading">Tonic</div>
+        <div class="canopy-container">
```
Comment (mahbubul14) [inline index.html:43]: - [ ] The project title ("Tonic") should use a header tag `hx` (Ex: `<h2>`, `<h3>` etc.) :+1:

File: index.html [hunk 5]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+            <div class="canopy-header">CANOPY</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">Back End Dev</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">2015</div>
```
Comment (mahbubul14) [inline index.html:48]: We use `<div>`s when you need to divide (that's what divs are supposed to be used for) sections of your page from one another. Header/footer from content for instance, or individual posts on a blog. However, you shouldn't use them when semantically another element would make sense.

- [ ] You can use `<ul>` and `<li>` in this section. :+1:

File: index.html [hunk 6]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+    </div>
+        <div class="tonic-container">
+            <div><img src="images/tonic2.png" alt="tonic"></div>
+            <div class="tonic-heading">Tonic</div>
+            <div class="canopy-container">
+                <div class="canopy-header">CANOPY</div>
+                <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                <div class="canopy-text">Back End Dev</div>
+                <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                <div class="canopy-text">2015</div>
```
Comment (mahbubul14) [inline index.html:103]: We use `<div>`s when you need to divide (that's what divs are supposed to be used for) sections of your page from one another. Header/footer from content for instance, or individual posts on a blog. However, you shouldn't use them when semantically another element would make sense.

- [ ] You can use `<ul>` and `<li>` in this section. :+1:

File: index.html [hunk 7]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+        </div>
+            <div class="tonic-container">
+                <div><img src="images/multi-post-stories2.png" alt="multi-post-stories2"></div>
+                <div class="tonic-heading">Multi-Post Stories</div>
+                <div class="canopy-container">
+                    <div class="canopy-header">CANOPY</div>
+                    <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                    <div class="canopy-text">Back End Dev</div>
+                    <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                    <div class="canopy-text">2015</div>
```
Comment (mahbubul14) [inline index.html:130]: We use `<div>`s when you need to divide (that's what divs are supposed to be used for) sections of your page from one another. Header/footer from content for instance, or individual posts on a blog. However, you shouldn't use them when semantically another element would make sense.

- [ ] You can use `<ul>` and `<li>` in this section. :+1:

File: index.html [hunk 8]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+        <div class="tonic-icons-container">
+            <ul>
+                <li><img src="images/html-tag.png" alt="html-tag"></li>
+                <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+           </ul>
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
```
Comment (mahbubul14) [inline index.html:65]: - [ ] Please follow the button styling: Active, Hover, and Pressed for the **"See Project"** button. :+1: 

b1

File: index.html [hunk 9]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+        <div class="tonic-icons-container">
+            <ul>
+              <li><img src="images/html-tag.png" alt="html-tag"></li>
+              <li><img src="images/css-tag.png" alt="css-tag"></li> 
+              <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+         </ul>
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
```
Comment (mahbubul14) [inline index.html:93]: - [ ] Please follow the button styling: Active, Hover, and Pressed for the **"See Project"** button. :+1: 

b1

File: index.html [hunk 10]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+            <div class="tonic-icons-container">
+                <ul>
+                    <li><img src="images/html-tag.png" alt="html-tag"></li>
+                    <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                    <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+               </ul>
+            </div>
+            <div class="tonic-btn">
+                <button type="button"> See Project</button>
+            </div>
```
Comment (mahbubul14) [inline index.html:120]: - [ ] Please follow the button styling: Active, Hover, and Pressed for the **"See Project"** button. :+1: 

b1

File: index.html [hunk 11]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+                <div class="tonic-icons-container">
+                    <ul>
+                        <li><img src="images/html-tag.png" alt="html-tag"></li>
+                        <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                        <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+                   </ul>
+                </div>
+                <div class="tonic-btn">
+                    <button type="button"> See Project</button>
+              </div>
```
Comment (mahbubul14) [inline index.html:147]: - [ ] Please follow the button styling: Active, Hover, and Pressed for the **"See Project"** button. :+1: 

b1

File: index.html [hunk 12]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+        <li><a href="#"><img src="images/twitter-icon.png" alt="twitter-icon"></a></li> 
+        <li><a href="#"><img src="images/linkedin-icon.png" alt="linkedin-icon"></a></li> 
+        <li><a href="#"><img src="images/vector-one.png" alt="vector-one"></a></li>
+        <li><a href="#"><img src="images/github-icon.png" alt="github-icon"></a></li> 
+        <li><a href="#"><img src="images/vector-two.png" alt="vector-two"></a></li> 
+    </ul>
+</div>
+<div class="tonic-btn">
+    <button type="button"> Get my resume</button>
+</div>
```
Comment (mahbubul14) [inline index.html:174]: - [ ] Please follow the button styling: Active, Hover, and Pressed for the **"Get my resume"** button. :+1: 

b1

File: index.html [hunk 13]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
...
+                    <button type="button"> See Project</button>
+              </div>
+            </div>
+                
+</section>
+<footer class="footer-container">
+    <h1>
+        About <br>
+        Myself
+    </h1>
```
Comment (mahbubul14) [inline index.html:155]: - [ ] Kindly remove the `footer` tag from here and use the `section` tag. "About Myself" section (line 151- line 201) should be wrapped in a `section` tag. :+1:

File: index.html [hunk 14]
```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
     </ul>
     </div>
 </section>
-  
+<section class="work-section">
+    <div class="tonic-container">
+        <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
```
Comment (mahbubul14) [inline index.html:41]: - [ ] Your page's background image is not matched with the Figma design. Please work on it. :+1: 

- [ ] The cards are not aligned properly. Kindly fix it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y1 | f1

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @godiarksel ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Please use media queries to make sure that your website displays correctly on desktop screens. Currently, you haven't used media queries for desktop.

- [ ]  Kindly use Flexbox to layout the elements on the page. :computer: 

- [ ] Please use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version. It is not currently optimized for the mobile version.

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] For readability, please add two spaces of indentation instead of using the tab key. You did it throughout your code in the `HTML` file. Please fix it. You can check[ HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 15]
```diff
@@ -69,11 +71,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
         <div><img src="images/multi-post-portfolio.png" alt="multi-post-portfolio"></div>
         <div class="tonic-heading">Multi-Post Stories</div>
```
Comment (zieeco) [inline index.html:72]: - [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`

File: index.html [hunk 16]
```diff
@@ -96,11 +100,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
             <div><img src="images/tonic2.png" alt="tonic"></div>
             <div class="tonic-heading">Tonic</div>
```
Comment (zieeco) [inline index.html:101]: - [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`

File: index.html [hunk 17]
```diff
@@ -123,11 +129,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
                 <div><img src="images/multi-post-stories2.png" alt="multi-post-stories2"></div>
                 <div class="tonic-heading">Multi-Post Stories</div>
```
Comment (zieeco) [inline index.html:130]: - [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`

File: index.html [hunk 18]
```diff
@@ -41,11 +41,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
         <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
         <div class="tonic-heading">Tonic</div>
```
Comment (zieeco) [inline index.html:42]: - [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @godiarksel, 



Good job on implementing the changes required by the previous reviewer!
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice design :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/a

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @godiarksel 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**